archai/nlp/nvidia_transformer_xl/mem_transformer.py

```diff
@@ -935,7 +941,7 @@ def forward(self, data:torch.Tensor, target:Optional[torch.Tensor], mems:Optiona
             loss = -F.log_softmax(logit, -1)[:, :, 0]
         else:
             loss, log_prob = self.crit(hidden=pred_hid.view(-1, pred_hid.size(-1)),
-                                       target=target.view(-1) if target is not None else None,
+                                       target=target.contiguous().view(-1) if target is not None else None,
```

Comment:
Please add comment why contiguous is needed.

Reply:
I have added a new commit with the following comment on why contiguous is needed:

"As we are transposing the `target`, we need it in a contiguous piece of memory before applying a tensor visualization (view)"